src/javascripts/Views/addMastersModal.js

```diff
@@ -0,0 +1,31 @@
+ngapp.controller('addMastersModalController', function($scope, $timeout, errorService, modalService) {
+    let opts = $scope.modalOptions;
+    let selectedFile = opts.handle;
+    let selectedFileIndex = xelib.GetFileLoadOrder(selectedFile);
+    let allMasters =xelib.GetLoadOrder().split('\r\n');
+    let availableMasters = allMasters.slice(0,selectedFileIndex).subtract(xelib.GetMasterNames(opts.handle));
```

Comment:
this line is a bit long, perhaps we can store the currentMasters in its own variable first?

also, perhaps we can refactor all of this logic into a xelib function in `masters.js` called get `GetAvailableMasters` or something similar, which would take a file handle and return an array of available master file names.